Sources/SwiftyCrop/View/CropView.swift

```diff
@@ -50,7 +51,7 @@ struct CropView: View {
 
         let dragGesture = DragGesture()
             .onChanged { value in
-                let maxOffsetPoint = viewModel.calculateDragGestureMax()
+                let maxOffsetPoint = maskShape == .rectangle ? viewModel.calculateDragGestureMaxRectangle() : viewModel.calculateDragGestureMax()
```

Comment:
(here)

If the func handles the mask shape you do not need to use a tenary here, makes it less complex and keeps logic inside the viewModel